agents/sbd/fence_sbd.py

```diff
@@ -15,6 +17,7 @@
 PATH_NOT_EXISTS = -1
 PATH_NOT_BLOCK = -2
 SBD_PID_FILE = "@SBDPID_PATH@"
+SBD_SYSCONFIG_FILE = "@INITCONFDIR@/sbd"
```

Comment:
Do we need it? See comment with config-file parser.

Reply:
Removed